compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/ExpectedTypeFinder.java

```diff
@@ -333,6 +347,57 @@ public Optional<TypeSymbol> transform(MethodCallExpressionNode node) {
             return Optional.empty();
         }
 
+//        TODO: Need to implement code to detect the position when the `PositionalArgumentNode`is null.
+//        (e.g. `arr.sort(array:ASCENDING, <cursor >`). Since sort is the only lang lib function that accepts a
+//        @typeParam function in the second parameter and it is yet to be supported, this should be implemented
+//        along with https://github.com/ballerina-platform/ballerina-lang/issues/41498.
```

Comment:
No need to include the entire link of the issue, we can just add the issue number (#41498).